src/main/java/com/fabriccommunity/spookytime/doomtree/DoomTree.java

```diff
@@ -0,0 +1,102 @@
+
+	public static Block MIASMA_BLOCK = Registry.register(Registry.BLOCK, SpookyTime.id("miasma"), new MiasmaBlock());
+	
+	public static Block DOOM_GLEAM_BLOCK = Registry.register(Registry.BLOCK, SpookyTime.id("doom_gleam"), new DoomGleamBlock());
+
+	public static Block DOOM_HEART_BLOCK = Registry.register(Registry.BLOCK, SpookyTime.id("doom_tree_heart"),
+			new DoomHeartBlock(FabricBlockSettings.of(Material.WOOD).dropsLike(DOOM_SAPLING).breakByTool(FabricToolTags.AXES, 3).strength(200.0F, 1200.0F).build()));
+
+	public static final BlockEntityType<DoomHeartBlockEntity> DOOM_HEART =
+			Registry.register(Registry.BLOCK_ENTITY, SpookyTime.id("doom_tree"), BlockEntityType.Builder.create(DoomHeartBlockEntity::new, DOOM_HEART_BLOCK).build(null));
+
+	public static Block PLACED_DOOM_LOG = SpookyBlocks.register("doom_log_p", new DoomLogBlock(logSettings().build(), true, 1));
+	public static Block PLACED_DOOM_LOG_CHANNEL = SpookyBlocks.register("doom_log_channel_p", new DoomLogBlock(logSettings().build(), true, 1));
+	public static Block PLACED_DOOM_LOG_TERMINAL = SpookyBlocks.register("doom_log_terminal_p", new DoomLogBlock(logSettings().build(), true, 1));
+
+	public static Block DOOM_LOG = Registry.register(Registry.BLOCK, SpookyTime.id("doom_log"),
+			new DoomLogBlock.Height(logSettings().dropsLike(PLACED_DOOM_LOG).build(), 0.04f));
+	public static Block DOOM_LOG_CHANNEL = Registry.register(Registry.BLOCK, SpookyTime.id("doom_log_channel"),
+			new DoomLogBlock.Height(logSettings().dropsLike(PLACED_DOOM_LOG_CHANNEL).build(), 0.02f));
+	public static Block DOOM_LOG_TERMINAL = Registry.register(Registry.BLOCK, SpookyTime.id("doom_log_terminal"),
+			new DoomLogBlock(logSettings().dropsLike(PLACED_DOOM_LOG_TERMINAL).build(), false, 0.02f));
+
+	public static Block DOOM_LEAF = SpookyBlocks.register("doom_leaves", new DoomLeafBlock(doomedSettings(), false, 1));
+
+	public static Block DOOMED_LOG = SpookyBlocks.register("doomed_log", new DoomedLogBlock(doomedSettings()));
+	public static Block DOOMED_EARTH = SpookyBlocks.register("doomed_earth", new DoomedBlock(doomedSettings()));
+	public static Block DOOMED_STONE = SpookyBlocks.register("doomed_stone", new DoomedBlock(doomedSettings()));
+	public static Block DOOMED_DUST = SpookyBlocks.register("doomed_dust", new DoomedBlock(doomedSettings()));
+	
+	public static Block DOOMED_RESIDUE_BLOCK = SpookyBlocks.register("doomed_residue_block", new Block(FabricBlockSettings.of(Material.EARTH).breakByHand(true).sounds(BlockSoundGroup.SAND).build()));
+	public static Block WARDING_ESSENCE_BLOCK = SpookyBlocks.register("warding_essence_block", new Block(FabricBlockSettings.of(Material.EARTH).breakByHand(true).sounds(BlockSoundGroup.SAND).build()));
+	
+	public static Item DOOMED_RESIDUE_ITEM = Registry.register(Registry.ITEM, SpookyTime.id("doomed_residue"), new Item((new Item.Settings()).group(SpookyTime.GROUP)));
```

Comment:
Why are you using `SpookyBlocks.register` for blocks but not using `SpookyItems.register` for items? Also we have a helper method in SpookyItems for getting a simple Item Settings.

Reply:
SpookyBlocks.register() is actually useful enough that I changed to make it public vs duplicate it.  All the other registration methods are protected so I didn't use 'em.